diff -Nur linux_logo-3.9b2.orig/logo_config linux_logo-3.9b2/logo_config
--- linux_logo-3.9b2.orig/logo_config	Sun Mar  4 22:55:55 2001
+++ linux_logo-3.9b2/logo_config	Mon Mar 26 20:48:07 2001
@@ -1,2 +1,9 @@
+./logos/distributions/pld.logo
 ./logos/banner.logo
 ./logos/classic.logo
+./logos/distributions/debian.logo
+./logos/distributions/debian-old.logo
+./logos/distributions/slackware.logo
+./logos/distributions/suse.logo
+./logos/alternate_oses/aix.logo
+./logos/alternate_oses/irix.logo
diff -Nur linux_logo-3.9b2.orig/logos/distributions/pld.logo linux_logo-3.9b2/logos/distributions/pld.logo
--- linux_logo-3.9b2.orig/logos/distributions/pld.logo	Thu Jan  1 01:00:00 1970
+++ linux_logo-3.9b2/logos/distributions/pld.logo	Mon Mar 26 20:46:50 2001
@@ -0,0 +1,39 @@
+# PLD logo by Michal Zawalich <michuz@pld-linux.org>
+
+
+SYSINFO_POSITION bottom
+
+DESCRIPTION_STRING PLD Linux banner
+
+
+BEGIN_ASCII_LOGO
+                               _______      ___          _______              
+            ###               /       `\   |   |        /       `\            
+           #o#o#             /______    |  |   |       /______    `\          
+           # V #              .----_)   |  |   |        .---. `\    |         
+          #     ##            |         /  |   |        |   |   |   |         
+         #       ##           |    ___/'   |   |______  |   |__/'   |         
+         Q#      #Q           |   |        |         /  |          /'         
+        QQQQ#   #QQQ          |___|        |________/   |________/'           
+         QQQ####QQQ                                                           
+                                                                              
+                             PLD Linux Distribution                           
+                        Homepage: http://www.pld-linux.org/                      
+END_ASCII_LOGO
+
+
+BEGIN_LOGO
+[0;0;30;36m                                _______      ___          _______              [1;37;40m
+[0;0;30;32m             ###[0;0;30;36m               /       `\   |   |        /       `\            [1;37;40m
+[0;0;30;32m            #[0;1;30;39mo[0;0;30;32m#[0;1;30;39mo[0;0;30;32m#[0;0;30;36m             /______    |  |   |       /______    `\          [1;37;40m
+[0;0;30;32m            # [0;0;30;36mV [0;0;30;32m#[0;0;30;36m              .----_)   |  |   |        .---. `\    |         [1;37;40m
+[0;0;30;32m           #     ##[0;0;30;36m            |         /  |   |        |   |   |   |         [1;37;40m
+[0;0;30;32m          #       ##[0;0;30;36m           |    ___/'   |   |______  |   |__/'   |         [1;37;40m
+[0;0;30;36m          Q[0;0;30;32m#      #[0;0;30;36mQ           |   |        |         /  |          /'         [1;37;40m
+[0;0;30;36m         QQQQ[0;0;30;32m#   #[0;0;30;36mQQQ          |___|        |________/   |________/'           [1;37;40m
+[0;0;30;36m          QQQ[0;0;30;32m####[0;0;30;36mQQQ                                                           [1;37;40m
+[0;0;30;32m                                                                              [1;37;40m
+[0;0;30;36m                             P[0;0;30;32mLD [0;0;30;36mL[0;0;30;32minux [0;0;30;36mD[0;0;30;32mistribution                           [1;37;40m
+[0;0;30;32m                        Homepage: http://www.pld-linux.org/                      [1;37;40m
+END_LOGO
+
