diff -Nur linux_logo-3.05/banner_logos/pld_ascii_banner.h linux_logo-3.05.new/banner_logos/pld_ascii_banner.h
--- linux_logo-3.05/banner_logos/pld_ascii_banner.h	Thu Jan  1 01:00:00 1970
+++ linux_logo-3.05.new/banner_logos/pld_ascii_banner.h	Thu Apr 27 00:29:43 2000
@@ -0,0 +1,15 @@
+/* by Michal Zawalich <michuz@pld.org.pl> */
+const char ascii_banner2[][400]={
+"%                               _______      ___          _______              ",
+"%            ###               /       `\\   |   |        /       `\\            ",
+"%           #o#o#             /______    |  |   |       /______    `\\          ",
+"%           # V #              .----_)   |  |   |        .---. `\\    |         ",
+"%          #     ##            |         /  |   |        |   |   |   |         ",
+"%         #       ##           |    ___/'   |   |______  |   |__/'   |         ",
+"%         Q#      #Q           |   |        |         /  |          /'         ",
+"%        QQQQ#   #QQQ          |___|        |________/   |________/'           ",
+"%         QQQ####QQQ                                                           ",
+"%                                                                              ",
+"%                         Polish(ed) Linux Distribution                        ",
+"%                        Homepage: http://www.pld.org.pl/                      "
+};
\ No newline at end of file
diff -Nur linux_logo-3.05/banner_logos/pld_banner.h linux_logo-3.05.new/banner_logos/pld_banner.h
--- linux_logo-3.05/banner_logos/pld_banner.h	Thu Jan  1 01:00:00 1970
+++ linux_logo-3.05.new/banner_logos/pld_banner.h	Thu Apr 27 00:39:18 2000
@@ -0,0 +1,15 @@
+/* by Michal Zawalich <michuz@pld.org.pl> */
+const char banner2[][400]={
+"[0;0;30;36m                                _______      ___          _______              [1;37;40m",
+"[0;0;30;32m             ###[0;0;30;36m               /       `\\   |   |        /       `\\            [1;37;40m",
+"[0;0;30;32m            #[0;1;30;39mo[0;0;30;32m#[0;1;30;39mo[0;0;30;32m#[0;0;30;36m             /______    |  |   |       /______    `\\          [1;37;40m",
+"[0;0;30;32m            # [0;0;30;36mV [0;0;30;32m#[0;0;30;36m              .----_)   |  |   |        .---. `\\    |         [1;37;40m",
+"[0;0;30;32m           #     ##[0;0;30;36m            |         /  |   |        |   |   |   |         [1;37;40m",
+"[0;0;30;32m          #       ##[0;0;30;36m           |    ___/'   |   |______  |   |__/'   |         [1;37;40m",
+"[0;0;30;36m          Q[0;0;30;32m#      #[0;0;30;36mQ           |   |        |         /  |          /'         [1;37;40m",
+"[0;0;30;36m         QQQQ[0;0;30;32m#   #[0;0;30;36mQQQ          |___|        |________/   |________/'           [1;37;40m",
+"[0;0;30;36m          QQQ[0;0;30;32m####[0;0;30;36mQQQ                                                           [1;37;40m",
+"[0;0;30;32m                                                                              [1;37;40m",
+"[0;0;30;36m                         P[0;0;30;32molish(ed) [0;0;30;36mL[0;0;30;32minux [0;0;30;36mD[0;0;30;32mistribution                        [1;37;40m",
+"[0;0;30;32m                        Homepage: http://www.pld.org.pl/                      [1;37;40m"
+};
diff -Nur linux_logo-3.05/defaults.h linux_logo-3.05.new/defaults.h
--- linux_logo-3.05/defaults.h	Wed Mar  1 01:16:13 2000
+++ linux_logo-3.05.new/defaults.h	Thu Apr 27 00:19:34 2000
@@ -12,6 +12,9 @@
 /* If a 0 is picked, classic mode (which some people prefer)will be default.*\
 \* You can over-ride this with the -banner and -classic switches.           */
 #define DEFAULT_BANNER_MODE 1
+
+/* by MZ */
+#define DEFAULT_PLD_MODE 1
 
 /* By setting this option to 1 you can have linux_logo always run in plain *\
 \* ascii mode.                                                             */
@@ -111,3 +114,9 @@
 
 /* This file contains the regular version of the banner logo */
 #include "./banner_logos/banner.h"
+
+/* by MZ */
+#include "./banner_logos/pld_ascii_banner.h"
+
+/* by MZ */
+#include "./banner_logos/pld_banner.h"
diff -Nur linux_logo-3.05/getsysinfo.h linux_logo-3.05.new/getsysinfo.h
--- linux_logo-3.05/getsysinfo.h	Tue Jun 22 17:35:07 1999
+++ linux_logo-3.05.new/getsysinfo.h	Thu Apr 27 00:19:34 2000
@@ -24,6 +24,7 @@
        int offset;
        int plain_ascii;
        int banner_mode;
+       int pld_mode;
        int wipe_screen;
        int show_uptime;
        int show_load;
diff -Nur linux_logo-3.05/linux_logo.c linux_logo-3.05.new/linux_logo.c
--- linux_logo-3.05/linux_logo.c	Thu Jan 13 19:44:36 2000
+++ linux_logo-3.05.new/linux_logo.c	Thu Apr 27 00:35:09 2000
@@ -48,6 +48,7 @@
     logo_info->offset=DEFAULT_OFFSET;                /* Defaults to 0    */
     logo_info->plain_ascii=DEFAULT_PLAIN_ASCII;      /* Defaults to No   */
     logo_info->banner_mode=DEFAULT_BANNER_MODE;      /* Defualts to Yes  */
+    logo_info->pld_mode=DEFAULT_PLD_MODE;	     /* Defaults to Yes	by MZ */
     logo_info->wipe_screen=DEFAULT_WIPE_SCREEN;      /* Defaults to No   */
     logo_info->show_uptime=DEFAULT_SHOW_UPTIME;      /* Defaults to No   */
     logo_info->show_load=DEFAULT_SHOW_LOAD;          /* Defaults to No   */
@@ -151,9 +152,11 @@
 	   "[-l]\n"
 	   "                    [-n] [-o num] [-p] [-rX] [-s] [-t str] [-u] "
 	   "[-v] [-w Num]\n"
-           "                    [-x] [-y] [-F format]\n",binname);
+           "                    [-x] [-y] [-F format] [-A] [-B]\n",binname);
     printf("         [-a]     -- Display logo as ascii only monochrome\n");
-    printf("         [-b]     -- New default Banner Logo!\n");
+    printf("         [-A]     -- PLD ascii logo\n");
+    printf("         [-b]     -- New Banner Logo!\n");
+    printf("         [-B]     -- PLD banner logo (default)\n");
     printf("         [-c]     -- The Old [classic] linux_logo look\n");
     printf("         [-d]     -- disable \"prettying\" of output\n");
     printf("         [-e file]-- Use \"file\" instead of /proc/cpuinfo [for "
@@ -294,10 +297,15 @@
     if (!logo_info->display_sysinfo_only) {
        for(i=0;i<12;i++) {
           do_spacing((logo_info->width-80)/2,logo_info->plain_ascii);
-          if (logo_info->plain_ascii) 
+          if ((logo_info->plain_ascii) && !(logo_info->pld_mode)) 
 	     ansi_print(ascii_banner[i],0,0,logo_info->symbol,' ',logo_info);
-          else ansi_print(banner[i],0,0,logo_info->symbol,
-			  logo_info->symbol_bgnd,logo_info);
+	  else if ((logo_info->plain_ascii) && (logo_info->pld_mode)) 
+	     ansi_print(ascii_banner2[i],0,0,logo_info->symbol,' ',logo_info);
+          else if (!(logo_info->pld_mode))
+	     ansi_print(banner[i],0,0,logo_info->symbol,
+		logo_info->symbol_bgnd,logo_info);
+	  else ansi_print(banner2[i],0,0,logo_info->symbol,
+	     logo_info->symbol_bgnd,logo_info);
           printf("\n");
        }
        if (!logo_info->display_logo_only) printf("\n");
@@ -331,10 +339,12 @@
     logo_info.cpuinfo_file=strdup("/proc/cpuinfo");
    
     while ((c = getopt (argc, argv,"F:"
-			           "a::b::c::de:fghk:lno:pr:st:uvw:xy"))!=-1)
+			           "a::A::b::B::c::de:fghk:lno:pr:st:uvw:xy"))!=-1)
        switch (c) {
-	  case 'a': logo_info.plain_ascii=1; break;
-	  case 'b': logo_info.banner_mode=1; break;
+	  case 'a': logo_info.plain_ascii=1; logo_info.pld_mode=0; break;
+	  case 'A': logo_info.plain_ascii=1; logo_info.pld_mode=1; break;
+	  case 'b': logo_info.banner_mode=1; logo_info.pld_mode=0; break;
+	  case 'B': logo_info.banner_mode=1; logo_info.pld_mode=1; break;
 	  case 'c': logo_info.banner_mode=0; break;
 	  case 'd': logo_info.pretty_output=0; break; 
 	  case 'e': logo_info.cpuinfo_file=strdup(optarg); break;
