--- linux_logo-4.04/linux_logo.h.orig	Sun Jul 14 20:40:59 2002
+++ linux_logo-4.04/linux_logo.h	Sat Aug 24 20:41:53 2002
@@ -15,6 +15,7 @@
        int display_usertext;
        int custom_format;
        int pretty_output;
+       int quote_logo_backslashes;
        int center_sysinfo;
        char *user_text;
        char *format;
--- linux_logo-4.04/linux_logo.c.orig	Fri Aug  9 18:50:49 2002
+++ linux_logo-4.04/linux_logo.c	Sat Aug 24 20:43:50 2002
@@ -196,6 +196,7 @@
     settings->display_sysinfo_only=0;
     settings->display_usertext=0;
     settings->custom_format=0;
+    settings->quote_logo_backslashes=0;		    /* Defaults to No   */
     settings->center_sysinfo=1;
     settings->user_text = NULL;  /* Change this and display_usertext to *\
                                  \*        have a default message       */
@@ -236,12 +237,14 @@
 
 
 
-char *print_line(char *string)
+char *print_line(char *string, struct linux_logo_info_type *settings)
 {
 
     char *temp_st=string;
 
     while ((*temp_st!='\n') && (*temp_st!='\0')) {
+       if('\\' == *temp_st && settings->quote_logo_backslashes)
+         putchar('\\');
        putchar(*temp_st);
        temp_st++;
     }
@@ -533,7 +536,7 @@
        if (settings->banner_mode) { /* Banner mode */
           for(i=0;i<ysize;i++) {
              shift_right((settings->width-80)/2,settings->plain_ascii);
-	     string_point=print_line(string_point);
+	     string_point=print_line(string_point, settings);
              printf("\n");
 	  }
           if (!settings->display_logo_only) printf("\n");
@@ -550,14 +553,14 @@
   
 	  for(i=0;i<7;i++) {
              shift_right(settings->offset,settings->plain_ascii);
-             string_point=print_line(string_point);
+             string_point=print_line(string_point, settings);
 	     printf("\n"); 
 	  }
 
              /* The next lines can have cpuinfo after them */
           for(i=7;i<ysize;i++) {
              shift_right(settings->offset,settings->plain_ascii);
-             string_point=print_line(string_point);
+             string_point=print_line(string_point, settings);
              shift_right(2,settings->plain_ascii);
              if (print_sysinfo(i-7,temp_string,settings)!=1) {
 	        if (!settings->plain_ascii) ansi_print("^[[1;37;40m");
@@ -737,7 +740,7 @@
              /*******************************************************/
        
        while ((c = getopt (*custom_argc, custom_argv,"D:F:L:"
-			           "a::b::c::de:fghiklno:pst:uvw:y"))!=-1) {
+			           "a::b::c::de:fghiklno:pqst:uvw:y"))!=-1) {
           switch (c) {
 	     case 'a': settings.plain_ascii=1; break;
 	     case 'b': settings.banner_mode=1; break;
@@ -813,6 +816,7 @@
 	               if ( endptr == optarg ) help_message(argv[0], 1);
 	               break;
 	     case 'p': settings.preserve_xy=1; break;
+	     case 'q': settings.quote_logo_backslashes=1; break;
 	     case 's': settings.skip_bogomips=1; break;
 	     case 't': 
 	               settings.display_usertext=1;
