diff -urN linux_logo-4.07.orig/vmw_string.c linux_logo-4.07/vmw_string.c
--- linux_logo-4.07.orig/vmw_string.c	2000-07-08 23:23:26.000000000 +0200
+++ linux_logo-4.07/vmw_string.c	2003-12-08 22:54:33.000000000 +0100
@@ -42,8 +42,10 @@
         strcpy(temp,"");
         ch=fgetc(fff);
         while ( (ch!='\n') && (ch!=EOF) && (i<((BUFSIZ-5)-1)) ) {
-	                   temp[i]=ch; i++;
-	                   ch=fgetc(fff);
+	                if ( ch != '\x0d' ){   
+			    temp[i]=ch; i++;
+			}
+			   ch=fgetc(fff);
 	}
         if(ch==EOF) return NULL;
         if (ch=='\n') {
