--- linux_logo-4.16/po/Makefile.orig	2007-01-27 23:00:32.000000000 +0100
+++ linux_logo-4.16/po/Makefile	2007-02-18 21:11:53.918103223 +0100
@@ -3,7 +3,7 @@
 INSTALLDIR=/usr/lib/locale
 PACKAGE = linux_logo
 
-CATALOGS = da.mo de.mo es.mo fr.mo nb.mo nl.mo it.mo pl.mo pt_BR.mo ru.mo sv.mo ua.mo zh_TW.mo
+CATALOGS = da.mo de.mo es.mo fr.mo nb.mo nl.mo it.mo pl.mo pt_BR.mo ru.mo sv.mo uk.mo zh_TW.mo
 
 POTFILES= ../*.c ../*.h
 
